.github/workflows/audit.yml

```diff
@@ -0,0 +1,22 @@
+# editorconfig-checker-disable-file
```

Comment:
rm

Reply:
Need to keep these on the workflow files as well. We don't control the format required by the actions api

Reply:
sorry im confused, what is the error? Seems like editorconfig should just verify that these are all correctly formatted.

Reply:
Yaml requires a specific format to know if things are a dictionary or part of a list etc. How we (and GitHub Actions) have defined our yaml configurations, doesn’t allow for always requiring 4 spaces. For example in the actions files having the “name” and “uses” keywords at the same indedentation depth depth is required but impossible with editorconfig because of the “-“ that needs to exist prior to the “name” keyword.

Reply:
Right.
Ok in that case I think we should change it. Can you add this to our .editorconfig?
```
# YAML Files
[*.{yml,yaml}]
indent_size = 2
```

Reply:
I think after doing that we can remove all of the editor-config-disable pragmas